pkg/agent/monitortool/latency_store.go

```diff
@@ -0,0 +1,242 @@
+// Copyright 2024 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package monitortool
+
+import (
+	"errors"
+	"net"
+	"sync"
+	"time"
+
+	"github.com/containernetworking/plugins/pkg/ip"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/util/k8s"
+)
+
+// LatencyStore is a store for latency information of connections between nodes.
```

Comment:
In the Antrea codebase, by convention, we capitalize all K8s-specific terms in comments and log messages. So node -> Node. It also applies to Pod, Service, etc.

Reply:
Thanks for the suggestion, I have modified it.